config/env/default.js

```diff
@@ -89,11 +89,16 @@ module.exports = {
 //					refresh: 8*3600000 // 8 Hours
 //				}
 //			},
-//			{
-//				file: '...',
-//				interval: 10000,
-//				config: {}
-//			}
+			{
+				file: './src/server/app/util/schedulers/inactivity-notification.server.service.js',
+				interval: 86400000, //every day
+				config: {
+					first: 30*86400000,  // 30 days
```

Comment:
What do you think about making these date thresholds an array so we're not tied to exactly 3 settings?